backend/Exceptions/SimpleMessageException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Exceptions;
+
+use ECSPrefix20210517\Nette\NotImplementedException;
+use Exception;
+use Throwable;
+
+class SimpleMessageException extends Exception
+{
+    protected const BAD_REQUEST = 400;
+    protected const METHOD_NOT_ALLOWED = 405;
+    protected const FORBIDDEN = 403;
+    protected const NOT_FOUND = 404;
+    protected const RANGE_NOT_SATISFIABLE = 416;
+    protected const IM_A_TEAPOT = 418;
+    protected const INTERNAL_SERVER_ERROR = 500;
```

Comment:
I am not sure that HTTP response codes will be useful there. Maybe let's create own error codes? Users won't execute this script themselfs so, these error codes should be usefull for developers, who will mantain this app.

Reply:
I think that maybe keeping error codes this way would be useful in the future. 

It's some form of standard and maybe in the future somebody would like to implement database to the project with Admin panel. Files with .csv extension were intended to be easy to modify. Maybe there will be a form in the future to add/modify company entries and errors could be thrown there?

I still need to properly assign error codes to the Exception as per Kamil's suggestions, because I'm not familiar yet which responses would be best in those scenarios.